dash_cytoscape/Cytoscape.py

```diff
@@ -54,7 +54,10 @@ class attribute).
         - `breadthfirst`: Tree structure built using BFS, with optional `roots`
         - `cose`: Force-directed physics simulation
 
-    2. The following external layouts are also included:
+    2. Some external layouts are also included. To use them, run
```

Comment:
Are you going to be moving these to a separate package at all in the future? (In the meantime, this is a great disclaimer :))

Reply:
I think it's simpler to keep them in the same package, since it's easier to maintain in the long term, and less confusing for the user. The overhead download usage might be something the users might worry about, in which case we could address when that happens.